src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -678,6 +679,21 @@ public Entity get(LightningStrikeEvent event) {
 				return event.getLightning();
 			}
 		}, 0);
+		// EndermanAttackPlayerEvent
+		if (Skript.classExists("com.destroystokyo.paper.event.entity.EndermanAttackPlayerEvent")) {
+			EventValues.registerEventValue(EndermanAttackPlayerEvent.class, Entity.class, new Getter<Entity, EndermanAttackPlayerEvent>() {
+				@Override
+				public Entity get(EndermanAttackPlayerEvent event) {
+					return event.getEntity();
```

Comment:
Have you tested this to ensure the event-entity is available and doesn't conflict with the event-player?

Reply:
I was able to set the enderman's name while sending messages to myself so I don't think there is a conflict between the two